free-science-books.md

```diff
@@ -96,7 +96,12 @@
 * [Lecture Notes on Thermodynamics](http://www.crectirupati.com/sites/default/files/lecture_notes/TD-lecture%20notes.pdf) - Munirathnam (PDF)
 * [Mathematical Methods in Physics - Course Notes](https://www.pa.ucla.edu/faculty-websites/dhoker-lecture-notes/graduate-courses/231A-lecture-notes-15.pdf) - Eric D’Hoker (PDF)
 * [Motion Mountain Physics](http://www.motionmountain.net) - Christoph Schiller
+* [NCERT Physics Part-1 | Class 11 | English](https://ncert.nic.in/textbook.php?keph1=1-8) (PDF)
```

Comment:
use link to the title page, and use that title. Attribution should come after the link:
`[Physics. Part I: Textbook for Class XI](https://ncert.nic.in/textbook.php?keph1=ps-8) - NCERT`
Similar for the other NCERT titles (you can spell out the words in NCERT, if you like.)